app/models/application_qualification.rb

```diff
@@ -60,6 +60,13 @@ class ApplicationQualification < ApplicationRecord
   validates :qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
   validates :non_uk_qualification_type, length: { maximum: MAX_QUALIFICATION_TYPE_LENGTH }, allow_blank: true
 
+  enum enic_reason: {
+    yes: 'yes',
+    waiting: 'waiting',
+    future: 'future',
+    no: 'no',
+  }
```

Comment:
```suggestion
  enum enic_reason: {
    yes: 'yes',
    waiting: 'waiting',
    future: 'future',
    no: 'no',
  }, prefix: true
```
How do you feel about adding this option - this would change the calls from something like `application_qualification.yes?` to `application_qualification.enic_reason_yes?`